packages/bpk-component-barchart/src/BpkChartDataTable.js

```diff
@@ -25,7 +27,15 @@ import STYLES from './BpkChartDataTable.scss';
 
 const getClassName = cssModules(STYLES);
 
-const BpkChartDataTable = props => {
+type Props = {
+  data: Array<any>,
```

Comment:
We typically try to avoid `any` unless absolutely necessary, is there something more specific we can use here?